.buildkite/pipeline.yml

```diff
@@ -23,3 +23,8 @@ steps:
     command: 'scripts/ci/nix-shell.sh --run scripts/ci/check-hydra.sh'
     agents:
       system: x86_64-linux
+
+  - label: 'bower2nix and node2nix'
```

Comment:
I'd be more specific here: `explorer frontend: bower2nix and node2nix`

Reply:
OK fixed.